.github/workflows/ReactNativeSlider-CI.yml

```diff
@@ -0,0 +1,161 @@
+name: ReactNativeSlider-CI
+
+on: [push]
+
+jobs:
+  install:
+    name: Install dependencies
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+
+      - name: Display tools versions
+        run: npm --version;
+
+      - name: Restore all the packages
+        run: npm install
+
+      - name: Cache
+        uses: actions/cache@v2.1.6
+        with:
+          path: ./package/node_modules
+          key: ${{ hashFiles('./package/package.json') }}
+
+
+  lint:
+    name: Lint the sources
+    runs-on: ubuntu-latest
+    needs: install
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Cache
+        uses: actions/cache@v2.1.6
+        with:
+          path: ./package/node_modules
+          key: ${{ hashFiles('./package/package.json') }}
+
+      - name: Run ESLint on the sources
+        run: cd package && npx eslint src
+
+
+  test:
+    name: Run the unit tests
+    runs-on: ubuntu-latest
+    needs: install
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Pull the npm dependencies
+        uses: actions/cache@v2.1.6
+        with:
+          path: ./package/node_modules
+          key: ${{ hashFiles('./package/package.json') }}
+
+      - name: Run unit tests
+        run: cd package && npx jest src
+
+
+  verify:
+    name: Verify the Example app sources
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run ESLint on the sources
+        run: cd example && npx eslint .
+
+
+  build-android-app:
+    name: Verify the package and example app builds for Android platform
+    runs-on: ubuntu-latest
+    needs: [verify]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Pull the npm dependencies
+        run: npm install
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Build the Android OS app
+        run: cd example/android && ./gradlew assembleDebug
+
+
+  build-iOS-app:
+    name: Verify the package and example app builds for iOS platform
+    runs-on: macos-latest
+    needs: [verify]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+
+      - name: Install node_modules on MacOS machine
+        run: npm install
+
+      - run: cd example/ios && xcode-select -p
```

Comment:
Do we need this step or it's an output for debugging?

Reply:
Debugging leftover, I'll remove that, thanks!